ext/ddtrace_profiling_native_extension/native_extension_helpers.rb

```diff
@@ -242,6 +247,15 @@ def self.pkg_config_missing?(command: $PKGCONFIG) # rubocop:disable Style/Global
           architecture_not_supported unless RUBY_PLATFORM.start_with?('x86_64', 'aarch64')
         end
 
+        private_class_method def self.on_ruby_2_1?
+          ruby_version_not_supported = explain_issue(
+            'your Ruby version (2.1) is too old to be supported.',
```

Comment:
I think the message is a bit negative, specially for clients that are not happy to be stuck on 2.1 themselves.

How about something like "the profiler only supports Ruby 2.2 or newer".

Reply:
Seems reasonable, changed in 91eb8628bdddba02ea4a2644c99824a040f3881b .